libjpeg-turbo.yaml

```diff
@@ -1,13 +1,13 @@
 package:
-  name: libjpeg
+  name: libjpeg-turbo
   version: 2.1.91
   epoch: 1
```

Comment:
The epoch should be bumped just to be sure.

Reply:
Done!